Adds ability to launch multiple jobs on cluster with different code changes (#832)
isaac-sim/IsaacLab: docs/source/deployment/cluster.rst

       and mounted into the singularity container. This should increase the speed of starting
       the simulation.
   * - CLUSTER_ISAACLAB_DIR
     - The directory on the cluster where the Isaac Lab logs are stored. This directory has to
       end on ``isaaclab``. It will be copied to the compute node and mounted into
       the singularity container. When a job is submitted, the latest local changes will

>        be copied to the cluster to a new directory in the format ``${CLUSTER_ISAACLAB_DIR}_${date}_${time}``

Reviewer: ```suggestion
       be copied to the cluster to a new directory in the format ``${CLUSTER_ISAACLAB_DIR}_${date_time}``
```

       with the date and time of the job submission. This allows to run multiple jobs with different code versions at
       the same time.
   * - CLUSTER_LOGIN
     - The login to the cluster. Typically, this is the user and cluster names,
       e.g., ``your_user@euler.ethz.ch``.
   * - CLUSTER_SIF_PATH
     - The path on the cluster where the singularity image will be stored. The image will be
       copied to the compute node but not uploaded again to the cluster when a job is submitted.
   * - REMOVE_CODE_COPY_AFTER_JOB

>      - Decide if the code copy should be removed after the job is finished. The logs from the job will not be deleted

Reviewer: ```suggestion
     - Whether the copied code should be removed after the job is finished or not. The logs from the job will not be deleted
```
